index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4dc3d7e55b.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/css2?family=Koulen&family=Luckiest+Guy&family=Poppins:wght@700&family=Special+Elite&family=Teko:wght@300&family=Yomogi&display=swap" rel="stylesheet">
+    <title>Josué</title>
+</head>
+<body>
+    <header class="Header">My Logo</header>
+    <nav class="navigation">
+    <button class="menu">
+    <i class="fa-solid fa-bars"></i></button>
+    </nav>
+    <section class="section">
+        <h1>I´m Josué Glad to see you!</h1>
+        
+    <div class="description">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    </div>
+    <div class="lets-connect">
+        <p>LET´S CONNECT</p>
+
+    </div>
+    
+    <div class="container">
+        <div class="icons">
+        <i class="fa-brands fa-twitter"></i>
+        <i class="fa-brands fa-google"></i>
+        <i class="fa-brands fa-facebook"></i>
+        <i class="fa-brands fa-whatsapp"></i>
+        <i class="fa-brands fa-github"></i>
+        <i class="fa-brands fa-instagram"></i>
+        <i class="fa-brands fa-linkedin-in"></i>
+        </div>
+    </div>
+    </section>
```

Comment:
Please stick to the Figma design provided as much as possible. Kindly, add `padding` to the left and right of this section to push the content a little to the right currently it's off to the left. Here is your design along with the Figma design:

| Your Design | Figma Design |
|-------------|---------------|
| <img width="340" alt="image" src="https://user-images.githubusercontent.com/34578631/171048055-aeac7e21-09e8-4269-97a4-2881690cff2a.png"> | <img width="370" alt="image" src="https://user-images.githubusercontent.com/34578631/171048100-1cce5f09-0315-4ad5-87a6-b779ca07fa35.png"> |